flocker/control/test/test_persistence.py

```diff
@@ -131,7 +139,53 @@ def test_file_is_created(self):
         """
         path = FilePath(self.mktemp())
         self.service(path)
-        self.assertTrue(path.child(b"current_configuration.v1.json").exists())
+        self.assertTrue(path.child(b"current_configuration.json").exists())
+
+    @validate_logging(assertHasAction, _LOG_UPGRADE, succeeded=True,
+                      startFields=dict(configuration=V1_TEST_DEPLOYMENT_JSON,
+                                       source_version=1, target_version=2))
```

Comment:
The `target_version` will change (and the test will break) when `_CONFIG_VERSION` changes.  Probably makes sense to put `_CONFIG_VERSION` into this assertion.